imod/mf6/multimodel/modelsplitter.py

```diff
@@ -86,6 +92,10 @@ def slice_model(partition_info: PartitionInfo, model: Modflow6Model) -> Modflow6
         new_idomain = model.domain.sel(coords).where(sliced_domain_2D, other=0)
 
     for pkg_name, package in model.items():
+        if isinstance(package, BoundaryCondition):
```

Comment:
Shouldn't this apply to all packages, not just the BoundaryCondition packages?

Reply:
The boundary conditions are special, because over a boundary condition water will enter or leave the model. Usually we assign a concentration to the boundary condition, which is used when water is entering the model over that boundary. When water leaves the model, it leaves with the concentration computed in the model. That's why the flow boundary conditions carry this concentration data if the flow model is used with transport. But the other packages (STO, NPF, DIS, etcetera) do not carry concentration data, and therefore don't need this operation.

Reply:
Isn't there a reason for those packages to have auxiliary variables for other reasons? Like the “ANGLDEGX” and “CDIST” auxiliary variables. Are those auxiliary variables propogates aswell? Or isnt that necesarry?

Reply:
Important here is: remove_**periodic**_auxiliary_variable, so these are aux variables which can change over time (if I'm not mistaken). This excludes the aux vars for the exchange packages. We might need to clarify that more in the method/function names.

Reply:
indeed, and in this PR I renamed _periodic_ to _transient_, because periodic somehow suggests a repeating pattern in the data.

Reply:
Good!